### PR TITLE
(DOCSP-45898): release notes v1.14.1

### PR DESCRIPTION
**Pull request by davidhou17**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kafka-connector/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45898 >
Staging - <https://deploy-preview-189--docs-kafka-connector.netlify.app/whats-new/>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -17,6 +17,14 @@ What's New
 .. meta::
    :keywords: bug fix, features
 
+.. _kafka-connector-whats-new-1.14.1:
+
+What's New in 1.14.1
+--------------------
+
+- Added support for the new Kafka SystemTime pattern, ``Time.SYSTEM``.
```
[the remark this answers is not in the document]
Comment (davidhou17) [reply source/whats-new.txt:25]: Good call, thanks!

[the remark this answers is not in the document]
Comment (Calvinnix) [reply source/whats-new.txt:25]: Oh sorry I should have clarified that we can remove `Added support for the new Kafka SystemTime pattern, ``Time.SYSTEM``. ` as well

[the remark this answers is not in the document]
Comment (davidhou17) [reply source/whats-new.txt]: Ah, fixed now

**Review by Calvinnix — APPROVED** [review]
LGTM! 🚀

**Review by kanchana-mongodb — APPROVED** [review]

**Comment by davidhou17** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.14|[](https://github.com/mongodb/docs-kafka-connector/pull/190)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**